pkg/apicapi/apic_types.go

```diff
@@ -490,7 +490,7 @@ func NewTagAnnotation(parentDn string, key string) ApicObject {
 	ret := newApicObject("tagAnnotation")
 	ret["tagAnnotation"].Attributes["key"] = key
 	ret["tagAnnotation"].Attributes["dn"] =
-		fmt.Sprintf("%s/annotationKey-%s", parentDn, key)
+		fmt.Sprintf("%s/annotationKey-[%s]", parentDn, key)
```

Comment:
Is this change backward compatible (with say 3.2)?

Reply:
While there is no functional impact, I went ahead and made this version dependent to handle it cleanly.

Reply:
Thanks. There was no functional impact to begin with. We still decided to make the change in favor of the newer release and but introduced the same issue in the older release. Given that we are just as likely to have the older release (3.2) deployed at least for some time, preserving backward compatibility makes sense.